server/sonar-server-common/src/main/java/org/sonar/server/rule/index/RuleIndex.java

```diff
@@ -153,6 +154,8 @@ public class RuleIndex {
   private final EsClient client;
   private final System2 system2;
 
+  private static final String COMM_LITERAL="activation";
```

Comment:
Same here. Just name it ACTIVATIOn_STR or something more descriptive.

Reply:
Modified